docs/index.rst

```diff
@@ -1,21 +1,39 @@
 Welcome to FMU-dataio's documentation
 =====================================
 
-FMU-dataio is a specialized library for managing data in Fast Model Update (FMU) workflows. It focuses on exporting data while adhering to the FMU standards, which include both file and folder conventions, and rich metadata integration for various data consumers. The library is designed for consistent usage across different stages of FMU workflows, such as pre- and post-processing jobs, and is compatible with Ensemble Reservoir Tool (ERT) FORWARD_MODEL, within and outside RMS.
+``fmu-dataio`` is a specialized Python library for managing data in Fast Model Update (FMU) workflows. 
+It focuses on exporting data while adhering to the FMU standards, which include both file
+and folder conventions, and rich metadata integration for various data consumers. The
+library is designed for consistent usage across different stages of FMU workflows,
+such as pre- and post-processing jobs, and is compatible with Ensemble Reservoir Tool (ERT)
+FORWARD_MODEL, within and outside RMS.
```

Comment:
This reads a bit awkward ("is compatible with ERT FORWARD_MODEL, within and outside RMS"). Not sure on a better wording;

> such as pre- and post-processing jobs from ERT and RMS workflows.

?

Reply:
Yes, I agree. I can rephrase that. It is remnant from existing text, where the main point probably was that it can be used both inside and outside RMS.

Reply:
Rephrased (happy to iterate further, don't think it's perfect)